common/models/user.js

```diff
@@ -50,6 +54,23 @@ module.exports = function(User) {
   // see:
   // https://github.com/strongloop/loopback/issues/1137#issuecomment-109200135
   delete User.validations.email;
+  // Updated workaround for email validation
+  // see:
+  // https://github.com/strongloop/loopback/issues/1137#issuecomment-271437926
+  User.validate('email', function(err) {
+    if (this.email && !isEmail(this.email)) {
+      // Here err is a callback
+      // See loopback api docs loopback-datasource-juggler/#validatable-validate
+      debug(err());
```

Comment:
What gets logged out from this debug statement? I can't find any return statement looking through the source